src/views/base.html

```diff
@@ -1,53 +1,46 @@
-      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-      ga('create', 'UA-2529322-33', 'auto');
-      ga('send', 'pageview');
-    </script>
-    {% endblock %}
-  </body>
+	<head>
+		<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+		<title>{% block pageTitle %}{{ title }}{% endblock %}</title>
+		<meta name="title" content="{{ title }}" />
+		<meta name="description" content="{{ description }}" />
+		<meta name=keywords content="{{ keywords }}">
+		<meta name=author content="Hack Cambridge Team">
+		<meta name="language" content="en">
+		<meta name="viewport" content="width=device-width, user-scalable=no" />
+		<meta property="og:title" content="{{ title }}" />
+		<meta property="og:url" content="{{ requestedUrl.href }}" />
+		<meta property="og:description" content="{{ description }}" />
+		<meta property="og:image" content="{{ requestedUrl.protocol }}//{{ requestedUrl.host }}{{ asset('images/fb-share.jpg') }}" />
+		<link rel="stylesheet" type="text/css" href="{{ asset('styles/hc-2018/base.css') }}">
+		<link rel="stylesheet" type="text/css" href="https://hackcambridgeassets.s3.amazonaws.com/fonts/PxGroteskRegular-Regular.css">
+		<link rel="stylesheet" type="text/css" href="https://hackcambridgeassets.s3.amazonaws.com/fonts/PxGroteskScreen-Regular.css">
+		<link rel="stylesheet" type="text/css" href="https://hackcambridgeassets.s3.amazonaws.com/fonts/PxGroteskLight-Regular.css">
+		<link rel="stylesheet" type="text/css" href="https://hackcambridgeassets.s3.amazonaws.com/fonts/PxGroteskBold-Regular.css">
+		<link rel="icon" href="{{ asset('images/hc-2018/favicons/favicon.png') }}" sizes="512x512" type="image/png">
+		<link rel="icon" href="{{ asset('images/hc-2018/favicons/favicon.ico') }}" sizes="16x16 24x24 32x32 48x48 64x64 96x96 128x128 256x256" type="image/vnd.microsoft.icon">
+		<link rel="icon" href="{{ asset('images/hc-2018/favicons/favicon.icns') }}" sizes="16x16 24x24 32x32 48x48 64x64 96x96 128x128 256x256 512x512">
+		<link rel="icon" href="{{ asset('images/hc-2018/favicons/favicon.svg') }}" sizes="any" type="image/svg+xml">
+		</head>
+	<body>
+		{% block content %}
+		{% endblock %}
+		
+		{% block scripts %}
+		<script type="text/javascript" src="{{ asset('scripts/main.js') }}"></script>
+		
+		<!-- Facebook Pixel Code -->
+		<script>
+			!function(f,b,e,v,n,t,s){if(f.fbq)return;n=f.fbq=function(){n.callMethod?n.callMethod.apply(n,arguments):n.queue.push(arguments)};if(!f._fbq)f._fbq=n;n.push=n;n.loaded=!0;n.version='2.0';n.queue=[];t=b.createElement(e);t.async=!0;t.src=v;s=b.getElementsByTagName(e)[0];s.parentNode.insertBefore(t,s)}(window,document,'script','//connect.facebook.net/en_US/fbevents.js');fbq('init', '412352858960781');fbq('track', "PageView");</script>
```

Comment:
Nit: please leave these split onto multiple lines so that each line of code is no longer than approx. 150 chars long.